nms/app/views/equipment/FEGGatewayTable.tsx

```diff
@@ -209,3 +244,103 @@ function GatewayStatusTable(props: WithAlert & {refresh: boolean}) {
   );
 }
 const StatusTable = withAlert(GatewayStatusTable);
+
+type EquipmentGatewayUpgradeType = {
+  name: string;
+  id: GatewayId;
+  hardwareId: string;
+  tier: string;
+  currentVersion: string;
+};
+function UpgradeTable() {
+  const ctx = useContext(GatewayTierContext);
+  const gwCtx = useContext(FEGGatewayContext);
```

Comment:
_optional:_ I would prefer a clearer naming of the elements, such that the distinction is clear on every usage.

Reply:
I agree

Reply:
If you agree why did you resolve this without changing the names 🤔

If you want to keep short names `tierCtx` and `gwCtx` would already be clearer.